quickwit/quickwit-doc-mapper/src/default_doc_mapper/field_mapping_entry.rs

```diff
@@ -225,6 +217,20 @@ impl BinaryFormat {
                 format!("expected hex string, got `{byte_str}`: {hex_decode_err}")
             })?,
         };
+        Ok(payload)
+    }
+
+    /// Parses the `serde_json::Value` into `tantivy::schema::Value`.
+    pub fn parse_json(&self, json_val: &JsonValue) -> Result<TantivyValue, String> {
```

Comment:
Comment and name suggest the method is more generic than it is
```suggestion
    /// Parses the `serde_json::Value::String` into `TantivyValue::Bytes`.
    pub fn parse_json_str(&self, json_val: &JsonValue) -> Result<TantivyValue, String> {
```

Reply:
ah I just saw it's a method in BinaryFormat

Reply:
I did not understand